exporter.go

```diff
@@ -31,8 +31,12 @@ import (
 	"github.com/prometheus/common/version"
 	"github.com/prometheus/exporter-toolkit/web"
 	webflag "github.com/prometheus/exporter-toolkit/web/kingpinflag"
+	"golang.org/x/sys/windows"
 )
 
+// https://learn.microsoft.com/en-us/windows/win32/procthread/process-security-and-access-rights
+const PROCESS_ALL_ACCESS = windows.STANDARD_RIGHTS_REQUIRED | windows.SYNCHRONIZE | 0xffff
```

Comment:
There is a const for 0xffff

https://github.com/golang/sys/blob/6943ab67c485d110829e8c77c3a46235a07d73b9/windows/security_windows.go#L1002

Reply:
Good spot, I've added that in